test/spec/modules/kargoBidAdapter_spec.js

```diff
@@ -63,13 +67,45 @@ describe('kargo adapter tests', function () {
       bids = [
         {
           params: {
-            placementId: 'foo'
+            placementId: 'foo',
+            socialCanvas: {
+              segments: ['segment_1', 'segment_2', 'segment_3'],
```

Comment:
@jsadwith @juliangan07 Sean and myself are unsure if these segments are strings or ints. Do you know which it is?

Reply:
Strings

Reply:
+1